firmware/mcal/stm32f767/periph/pwm.h

```diff
@@ -60,10 +60,22 @@ class PWMOutput : public shared::periph::PWMOutput {
         return float(pulse) / float(period) * 100.0f;
     }
 
+    void SetFrequency(float frequency) override {
+        frequency_ = std::max(0.000015259022f, frequency);
```

Comment:
make this minimum a named constant to express why we have it and where it comes from `kMinimumFrequency = 0.00...`